docs/docker-stack/build.rst

```diff
@@ -180,7 +180,7 @@ In the simplest case building your image consists of those steps:
    In case you use some kind of registry where you will be using the image from, it is usually named
    in the form of ``registry/image-name``. The name of the image has to be configured for the deployment
    method your image will be deployed. This can be set for example as image name in the
-   `docker-compose file <running-airflow-in-docker>`_ or in the `Helm chart <helm-chart>`_.
+   `docker-compose file <https://airflow.apache.org/docs/apache-airflow/stable/howto/docker-compose/index.html>`_ or in the `Helm chart <https://airflow.apache.org/docs/helm-chart/stable/index.html>`_.
```

Comment:
Actually - those should Sphinx directives not URLs. Something like that (but you need to build the docs locally with `breeze build-docs` or let CI do it for you and verify if the links are correct"

```suggestion
   :doc:`apache-airflow:howto/docker-compose/index` or in the :doc:`helm-chart:index>`.
```

Except automatically generating the right link - also in the local documentation, this has the added advantage that in case of documentation refactorings, Sphinx will automatically check if the links are still valid and will fail docs building if they are not

Reply:
Thanks for the comment @potiuk. Adding these changes now and letting the CI go ahead with it